### ISSUE TITLE
Setting tree_root_id hides all contexts

### ISSUE THREAD

**Issue by iamdave210** [modxcms/revolution#13490]
### Summary
If tree_root_id is set on a user account that has access to multiple contexts only the context with the specified resource will be visible.

### Step to reproduce
Create multiple contexts with some resources and a manager user with access said contexts. Set tree_root_id for said user to some ID from one of the contexts. Log in as that user.

### Observed behavior
Contexts not containing the specified resource will be hidden entirely.

### Expected behavior
Contexts not containing the specified resource should be unmodified. Also, it would be nice to be able to enter a comma-separated list of IDs in order to set tree_root_id for multiple contexts (since resource IDs are unique).

### Environment
MODX 2.5.4, Apache 2.2.15, MySQL 5.5.56, all browsers

**Comment by wuuti** [comment]
@iamdave210 you should be able to set tree_root_id per context in the context settings to support multiple contexts. 

I am not sure that your observed behaviour can be seen as a bug here...

**Comment by rtripault** [comment]
Agreed, this setting is somehow one that won't work properly "per user" when on a multi contexts setup.
I see the need for it tho... so that's probably more of a feature request.

I guess one option could be to allow a setting per context, ie. `{context-key}_tree_root_id`

**Comment by gpsietzema** [comment]
#modxbughunt #1point to @wuuti

**Comment by iamdave210** [comment]
Yeah, maybe more of a feature request. I can't think of a use case for setting it on the context tho. The situation where I've needed it is making access to sections of the site more granular for users. Many of my users don't need access to the whole context, only a part of it.
